packages/netlify-cms-core/src/backend.ts

```diff
@@ -84,25 +84,83 @@ function getEntryBackupKey(collectionName?: string, slug?: string) {
   return `${baseKey}.${collectionName}${suffix}`;
 }
 
+const getEntryField = (field: string, entry: EntryValue) => {
+  const nestedFields = field.split('.');
+  let value = entry.data;
+  for (let i = 0; i < nestedFields.length; i++) {
```

Comment:
Cleaner to use a  lodash `get` here instead of the loop.

Reply:
Agree, I just extracted the common logic that was already used.
I'll update it